### DISCUSSION TITLE
Add number visualisation for difficulty settings

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#28520] [Ideas] [answered]
have to rely on tooltips which is tedious

![Screenshot 2024-06-18 151536](https://github.com/ppy/osu/assets/30382015/ce6ec45b-7f62-4032-a544-b6fc8530e6f4)

**Comment by bdach** [comment, marked as answer]
Whole screen is on my list to be redone and that is one of the items to be addressed there so
